### DISCUSSION TITLE
Is there a Arabic language support for Win Merge

### DISCUSSION THREAD

**Discussion by fhobeidat** [WinMerge/winmerge#1534] [Q&A]
WinMerge is an excellent software that make it very easy to compare text between different text files, but unfortunately when it comes to Arabic the format becomes unreadable. Is there a Arabic language plugin that can make reading the Arabic text possible?

**Comment by sdottaka** [comment]
Unfortunately, the text editor component used by WinMerge does not support the Arabic language.
Pull requests are welcome.
